projectforum/projects/tests/test_project_filter.py

```diff
@@ -0,0 +1,186 @@
+        self.assertEqual(0, len(contents['projects']))
+
+    def test_list_projects_by_title_down(self):
+        test_create_projects.create_many_projects()
+        response = project_filters.get_projects(status=1,
+                                                keywords='',
+                                                order='title',
+                                                salary='lump',
+                                                ascending=False)
+        contents = response
+        self.assertEqual('F', contents[0].title)
+        self.assertEqual('A', contents[5].title)
+        self.assertEqual('C', contents[3].title)
+
+    def test_list_projects_by_title_up(self):
+        test_create_projects.create_many_projects()
+        response = project_filters.get_projects(status=1,
+                                                keywords='',
+                                                order='title',
+                                                salary='lump',
+                                                ascending=True)
+        contents = response
+        self.assertEqual('A', contents[0].title)
+        self.assertEqual('F', contents[5].title)
+        self.assertEqual('D', contents[3].title)
+
+    def test_list_projects_by_newest(self):
+        test_create_projects.create_many_projects()
+        response = project_filters.get_projects(status=1,
+                                                keywords='',
+                                                order='timestamp',
+                                                salary='lump',
+                                                ascending=False)
+        contents = response
+        self.assertEqual('B', contents[0].title)
+        self.assertEqual('E', contents[5].title)
+        self.assertEqual('D', contents[3].title)
+
+    def test_list_projects_by_oldest(self):
+        test_create_projects.create_many_projects()
+        response = project_filters.get_projects(status=1,
+                                                keywords='',
+                                                order='timestamp',
+                                                salary='lump',
+                                                ascending=True)
+        contents = response
+        self.assertEqual('E', contents[0].title)
+        self.assertEqual('B', contents[5].title)
+        self.assertEqual('C', contents[3].title)
+
+    def test_list_projects_by_pay_lump_down(self):
+        test_create_projects.create_many_projects()
+        response = project_filters.get_projects(status=1,
+                                                keywords='',
+                                                order='payment',
+                                                salary='lump',
+                                                ascending=False)
+        contents = response
+        self.assertEqual('D', contents[0].title)
+        self.assertEqual('B', contents[5].title)
```

Comment:
Ok, that's fine. Just making sure it was intended functionality :) Seems like a good solution for ordering.